adal/src/main/java/com/microsoft/aad/adal/AcquireTokenSilentHandler.java

```diff
@@ -118,6 +118,78 @@ AuthenticationResult getAccessToken() throws AuthenticationException {
         return AuthenticationResult.createResult(accessTokenItem);
     }
 
+    AuthenticationResult getAccessTokenUsingAssertion() throws AuthenticationException {
+        final String methodName = ":getAccessTokenUsingAssertion";
+        final AuthenticationResult result = acquireTokenWithAssertion();
+        
+        if (result != null && !StringExtensions.isNullOrBlank(result.getAccessToken())) {
+            try {
+                mTokenCacheAccessor.updateTokenCache(mAuthRequest, result);
+            } catch (MalformedURLException e) {
+                Logger.v(TAG + methodName, "Access token fetched but unable to update token cache");
```

Comment:
Let's log this error as a warning, instead of as verbose

Reply:
Sure, will change that